.github/workflows/windows-build-test.yml

```diff
@@ -57,4 +57,4 @@ jobs:
 
       - name: Build app
         working-directory: ${{ matrix.working-directory }}/windows
-        run: npx react-native run-windows --logging --no-packager --no-deploy
\ No newline at end of file
+        run: npx react-native run-windows --logging --no-packager --no-deploy --no-autolink
```

Comment:
Why we're disabling autolinking on windows CI?

Reply:
Without that `--no-autolink` new libraries will be linking on the Windows platform. 
We do not want to install react-native-view-shot on the Windows platform.

Reply:
Doesn't it mean no autolinking at all? Shouldn't we link other libraries manually in that case?

Reply:
Any of the new libraries aren't needed in the Windows build test.